counterexamples/transportation/segment/road/bad-road-invalid-min-occupancy-value.json

```diff
@@ -0,0 +1,37 @@
+{
+    "id": "invalidMinOccupancyValue",
+    "type": "Feature",
+    "geometry": {
+      "type": "LineString",
+      "coordinates": [[2, 2], [3, 3]]
+    },
+    "properties": {
+      "theme": "transportation",
+      "type": "segment",
+      "version": 19,
+      "update_time": "2023-02-23T00:04:00-08:00",
+      "subtype": "road",
+      "road": {
+        "flags": [
+          {
+            "values": ["is_tunnel"]
+          }
+        ],
+        "lanes": [
+            {
+                "restrictions": {
+                    "min_occupancy": [
+                        {
+                            "value": 0
+                        }
+                    ]
```

Comment:
Sorry for one further change, but I can't think of a good reason why `min_occupancy` shouldn't be a flat value. IMO we should just make it a flat integer with a minimum value of 1.